### PR TITLE
API: Universal swagger generation

### PR DESCRIPTION
**Pull request by papagian**
**What this PR does / why we need it**:
It changes the Makefile targets for generating the Grafana and alerting swagger specifications using [bingo](https://github.com/bwplotka/bingo) so that it is always used the same binary regardless of the underlying operating system.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: pkg/services/ngalert/api/tooling/Makefile [hunk 1]
```diff
@@ -21,21 +23,7 @@ ifneq ($(OS),Windows_NT)
...
-		quay.io/goswagger/swagger:$(SWAGGER_TAG) \
-		-c \
-		"swagger generate spec -m -o $@; swagger generate spec -m --include-tag=stable -o spec-stable.json"
-
-ensure_go-swagger_mac:
-	@hash swagger &>/dev/null || (brew tap go-swagger/go-swagger && brew install go-swagger)
-
-spec.json-mac: ensure_go-swagger_mac $(GO_PKG_FILES)
-	swagger generate spec -m -w $(API_DIR) -o spec.json && swagger generate spec -m --include-tag=stable -o ngalert.json
+	SWAGGER_GENERATE_EXTENSION=false $(SWAGGER) generate spec -m -w $(API_DIR) -o spec.json && $(SWAGGER) generate spec -m --include-tag=stable -o spec-stable.json
```
Comment (papagian) [inline pkg/services/ngalert/api/tooling/Makefile:26]: `SWAGGER_GENERATE_EXTENSION=false` is for not including the `x-go-name` properties in the output.

**Review by ying-jeanne — APPROVED** [review]
Tested and Works, I might have a local environment that is massed up a bit, I had to export GOBIN=`pwd`/.bingo, then source the .env file to have a right path to the cached swagger. nice work 🎉

**Comment by gotjosh** [comment]
Tagged @alexweav so he can review when he's back.

**Comment by papagian** [comment]
@grafana/alerting-squad I have merged this because it was blocking (due to conflicts) for some other work; if you encounter any issues with the alerting specification generation because of this let me know.

**Pull request merged**